fix.py

```diff
@@ -30,168 +33,158 @@ def game_id():
+            return title
+        except OSError:# as e:
+            #log.info('OSError occurred: {}'.format(e))  # used for debugging
+            pass
+        except IndexError:# as e:
+            #log.info('IndexError occurred: {}'.format(e))  # used for debugging
+            pass
+        except UnicodeDecodeError:# as e:
+            #log.info('UnicodeDecodeError occurred: {}'.format(e))  # used for debugging
+            pass
     else:
         try:
             game_library = re.findall(r'.*/steamapps', os.environ['PWD'], re.IGNORECASE)[-1]
-            game_manifest = os.path.join(game_library, 'appmanifest_' + game_id() + '.acf')
+            game_manifest = os.path.join(game_library, f'appmanifest_{get_game_id()}.acf')
 
             with io.open(game_manifest, 'r', encoding='utf-8') as appmanifest:
                 for xline in appmanifest.readlines():
                     if 'name' in xline.strip():
                         name = re.findall(r'"[^"]+"', xline, re.UNICODE)[-1]
                         return name
         except OSError:
-            return 'UNKNOWN'
+            pass
         except IndexError:
-            return 'UNKNOWN'
+            pass
         except UnicodeDecodeError:
-            return 'UNKNOWN'
+            pass
+
     return 'UNKNOWN'
 
 
-def run_fix(gameid):
-    """ Loads a gamefix module by it's gameid
+def get_store_name(store: str) -> str:
+    """ Mapping for store identifier to store name
+    """
+    return {
+        'amazon': 'Amazon',
+        'battlenet': 'Battle.net',
+        'ea': 'EA',
+        'egs': 'EGS',
+        'gog': 'GOG',
+        'humble': 'Humble',
+        'itchio': 'Itch.io',
+        'steam': 'Steam',
+        'ubisoft': 'Ubisoft',
+        'zoomplatform': 'ZOOM Platform'
+    }.get(store, None)
+
+
+def get_module_name(game_id: str, default: bool = False, local: bool = False) -> str:
+    """ Creates the name of a gamefix module, which can be imported
+    """
+    if os.environ.get('STORE'):
+        store = os.environ['STORE'].lower()
+    elif game_id.isnumeric():
+        store = 'steam'
+
+    if store != 'steam':
+        log.info(f'Non-steam game {get_game_name()} ({game_id})')
+
+        store_name = get_store_name(store)
+        if store_name:
+            log.info(f'{store_name} store specified, using {store_name} database')
+        else:
+            log.info('No store specified, using UMU database')
+            store = 'umu'
+
+    return (f'protonfixes.gamefixes-{store}.' if not local else 'localfixes.') +\
+           (game_id if not default else 'default')
+
+
+def _run_fix_local(game_id: str, default: bool = False) -> bool:
+    """ Check if a local gamefix is available first and run it
     """
+    localpath = os.path.expanduser('~/.config/protonfixes/localfixes')
+    module_name =  game_id if not default else 'default'
+
+    # Check if local gamefix exists
+    if not os.path.isfile(os.path.join(localpath, module_name + '.py')):
```

Comment:
`f'{module_name}.py'`

Reply:
There are more of these, I updated strings that are formatted or consist of multiple concatenations.
I left strings that are simple prefixes or suffixes.

I could change them too, and we could also enforce them through the linter.